back/src/Guide/Database/Convert.hs

```diff
@@ -0,0 +1,173 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+
+module Guide.Database.Convert
+       (
+       -- Encoders
```

Comment:
Export list sections should look like `-- * Encoders`, `-- ** Encoders`, etc. Otherwise they won't show up in Haddock.